src/System.Management.Automation/commands/utility/FormatAndOutput/common/DefaultFormatters/DotNetTypes_format_ps1xml.cs

```diff
@@ -534,16 +534,16 @@ private static IEnumerable<FormatViewDefinition> ViewsOf_System_Diagnostics_Proc
             yield return new FormatViewDefinition("process",
                 TableControl.Create()
                     .AddHeader(Alignment.Right, label: "NPM(K)", width: 7)
-                    .AddHeader(Alignment.Right, label: "PM(K)", width: 8)
-                    .AddHeader(Alignment.Right, label: "WS(K)", width: 10)
+                    .AddHeader(Alignment.Right, label: "PM(M)", width: 8)
+                    .AddHeader(Alignment.Right, label: "WS(M)", width: 10)
                     .AddHeader(Alignment.Right, label: "CPU(s)", width: 10)
                     .AddHeader(Alignment.Right, width: 6)
                     .AddHeader(Alignment.Right, width: 3)
                     .AddHeader()
                     .StartRowDefinition()
                         .AddScriptBlockColumn("[long]($_.NPM / 1024)")
-                        .AddScriptBlockColumn("[long]($_.PM / 1024)")
-                        .AddScriptBlockColumn("[long]($_.WS / 1024)")
+                        .AddScriptBlockColumn("\"{0:N3}\" -f [float]($_.PM / (1024 * 1024))")
```

Comment:
I suppose seeing `0.00` looks wrong and some might investigate, that extra character is also causing problems - here is an example that I just tested with your changes:

```
   2663 ...6.016      0.004             11940   0 vmmem
   2655 ...6.016    234.953             12720   0 vmmem
```

And to me - this is worse - I worry more about processes using a lot of memory than those using little memory.